protos/mission/mission.proto

```diff
@@ -221,3 +249,17 @@ message MissionResult {
     Result result = 1; // Result enum value
     string result_str = 2; // Human-readable English string describing the result
 }
+
+// Progress data coming from mission upload, in percent.
+message ProgressData {
+    float progress = 1 [(mavsdk.options.default_value)="NaN"]; // Progress (percentage)
```

Comment:
Percentage would mean 0..100 but I'd probably just do 0..1.0, right?

Reply:
This is the same as the calibration, but I really don't mind. I'll change it to `Progress (0..1.0)` :+1: